[2024-05-09T10:10:14.046+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-09T10:10:14.219+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mysql_table_dump.mysql_table_dump_task manual__2024-05-09T10:10:11.040782+00:00 [queued]>
[2024-05-09T10:10:14.235+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mysql_table_dump.mysql_table_dump_task manual__2024-05-09T10:10:11.040782+00:00 [queued]>
[2024-05-09T10:10:14.235+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-09T10:10:14.261+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): mysql_table_dump_task> on 2024-05-09 10:10:11.040782+00:00
[2024-05-09T10:10:14.273+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'mysql_table_dump', 'mysql_table_dump_task', 'manual__2024-05-09T10:10:11.040782+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/dump_load_dag.py', '--cfg-path', '/tmp/tmpc8zre0ea']
[2024-05-09T10:10:14.277+0000] {standard_task_runner.py:91} INFO - Job 10: Subtask mysql_table_dump_task
[2024-05-09T10:10:14.277+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=226) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-09T10:10:14.278+0000] {standard_task_runner.py:63} INFO - Started process 227 to run task
[2024-05-09T10:10:14.395+0000] {task_command.py:426} INFO - Running <TaskInstance: mysql_table_dump.mysql_table_dump_task manual__2024-05-09T10:10:11.040782+00:00 [running]> on host 808b91199023
[2024-05-09T10:10:14.688+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mysql_table_dump' AIRFLOW_CTX_TASK_ID='mysql_table_dump_task' AIRFLOW_CTX_EXECUTION_DATE='2024-05-09T10:10:11.040782+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-09T10:10:11.040782+00:00'
[2024-05-09T10:10:14.691+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-09T10:10:14.775+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-09T10:10:14.780+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'mysqldump -u root -proot -h host.docker.internal -P 3306 fc_facts fc_deposit_facts > /opt/***/dags/dump_file.sql 2>/opt/***/dags/dump_error.log']
[2024-05-09T10:10:14.794+0000] {subprocess.py:86} INFO - Output:
[2024-05-09T10:10:15.453+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-09T10:10:15.457+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-09T10:10:15.511+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=mysql_table_dump, task_id=mysql_table_dump_task, execution_date=20240509T101011, start_date=20240509T101014, end_date=20240509T101015
[2024-05-09T10:10:15.544+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-09T10:10:15.580+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-09T10:10:15.583+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
